startup_plugin.sh

```diff
@@ -0,0 +1,7 @@
+vault login root
+vault secrets enable -path=kubernetes-secrets-reader vault-plugin-secrets-kubernetes-reader
```

Comment:
Future: if you ever want this plugin to support multicluster then this won't really make sense as you would want to enable it in multiple paths. We don't have such a requirement though.

Reply:
Thats not correct. the auth path should be enabled for each cluster but the backend only once.